### DISCUSSION TITLE
Ability to configure lambda function retry attempts

### DISCUSSION THREAD

**Discussion by filmaj** [architect/architect#1277] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
I would like to configure the "retry attempts" configuration option available on Lambda functions; it seems like the default is always set to 2.

**Describe the solution you'd like**
Maybe an option via the arc-config file?

**Describe alternatives you've considered**
Writing my own macro for this (yet to do it)

**Comment by ryanblock** [comment]
++

**Comment by brianleroux** [comment]
Do not think this is really possible or desirable; different event sources have different retry semantics and that's probably how you want it to be. SNS topics retry three times (which can be configured yes), SQS will retry until the heat death of the universe (also configurable but herein are dragons), APIG is synchronous (and per HTTP idempotent) so there are no retries. Etc.
